addons/mail/models/ir_attachment.py

```diff
@@ -93,6 +100,8 @@ def _to_store_defaults(self, target):
             Store.One("thread", [], as_thread=True),
             "type",
             "url",
+            "has_thumbnail",
+            Store.Attr("thumbnail_access_token", lambda a: a._get_thumbnail_token()),
```

Comment:
alphabetical order (unless necessary for query count, which shouldn't be the case here)